homeassistant/components/hunterdouglas_powerview/scene.py

```diff
@@ -47,10 +53,21 @@ def __init__(
     ) -> None:
         """Initialize the scene."""
         super().__init__(coordinator, device_info, room_name, scene.id)
-        self._scene = scene
+        self._scene: PvScene = scene
         self._attr_name = scene.name
         self._attr_extra_state_attributes = {STATE_ATTRIBUTE_ROOM_NAME: room_name}
+        self._forced_resync: list = []
+
+    async def _async_force_resync(self, *_: Any) -> None:
+        """Force a resync after an update since the hub may have stale state."""
+        for shade_id in self._forced_resync:
+            _LOGGER.debug("Force resync of shade %s", self.name)
+            await self.data.get_shade(shade_id=shade_id).refresh()
+        self._forced_resync = []
```

Comment:
Let's pull this out for now. Once we merge everything this can be a future improvement